src/content-helper/editor-sidebar/class-editor-sidebar.php

```diff
@@ -48,7 +49,8 @@ public function __construct( Parsely $parsely ) {
 
 		// Instantiate the features.
 		$this->features = array(
-			'Smart_Linking' => new Smart_Linking( $this ),
+			'Smart_Linking'     => new Smart_Linking( $this ),
+			'Excerpt_Generator' => new Excerpt_Suggestions( $this ),
```

Comment:
This is a fair point, I'm not sure if we kept the same name because of #2811 (will look now), but I agree we should change this, and possibly the `Endpoint_Excerpt_Generator` class from the Suggestions API service endpoint. What do you think?

Reply:
I agree, let's keep it as is it, and we can open a new PR to make it consistent later on.